components/x-follow-button/src/FollowButton.jsx

```diff
@@ -1,8 +1,16 @@
 import h from '@financial-times/x-engine';
+import {withActions} from '@financial-times/x-interaction';
 
 import Input from './Input';
 import Button from './Button';
 
+const followButtonActions = withActions(() => ({
+	onClickAction(props, event) {
+		event.preventDefault();
+		console.log('follow button pressed');
```

Comment:
Why do we need to `console.log` here and if yes why not to use smth like `n-logger` here?